### PR TITLE
ephys updates

### PR DESCRIPTION
**Pull request by saskiad**
updating ephys schema based on feedback from Josh

### CODE REVIEW

**Comment by saskiad** [comment]
also closes issue #106

File: src/aind_data_schema/ephys/ephys_rig.py [hunk 1]
```diff
@@ -23,6 +23,11 @@ class HarpDevice(Enum):
     INPUT_EXPANDER = "Input Expander"
 
 
+class HarpDevice(Device):
+    """Describes a Harp device"""
+    name: HarpDeviceName = Field(..., title="Name")
+    device_version: str = Field(..., title="Device version")
```
Comment (dyf) [inline src/aind_data_schema/ephys/ephys_rig.py:29]: do we want version on top level `Device`?

File: src/aind_data_schema/ephys/ephys_rig.py [hunk 2]
```diff
@@ -176,6 +215,23 @@ class EphysProbe(Device):
...
+    probe_manipulator: Manipulator = Field(..., "Manipulator")
+    calibration_data: str = Field(
+        ..., title="Calibration data", 
+        description="Path to calibration data"
+    )
+    calibration_date: Optional[datetime] = Field(
+        None, title="Calibration date"
+    )
+
+class DAQ(Device):
```
Comment (dyf) [inline src/aind_data_schema/ephys/ephys_rig.py:227]: I think this is identical to the `DAQ` in imaging.  Move this definition to `device.py` and share?

**Review by dyf — CHANGES_REQUESTED** [review]
Couple small things.

File: src/aind_data_schema/ephys/ephys_rig.py (hunk 1 above)
Comment (saskiad) [reply src/aind_data_schema/ephys/ephys_rig.py:30]: I don't think so... I think for most devices this shows up as model numbers. Not sure why they are version numbers for Harp

File: src/aind_data_schema/ephys/ephys_rig.py (hunk 2 above)
Comment (saskiad) [reply src/aind_data_schema/ephys/ephys_rig.py:233]: yes

File: src/aind_data_schema/ephys/ephys_rig.py (hunk 1 above)
Comment (dyf) [reply src/aind_data_schema/ephys/ephys_rig.py:30]: bueno.

**Review by dyf — APPROVED** [review]

**Pull request merged**